nodes/src/nodes/llm_vision_gemini/IInstance.py

```
@@ -0,0 +1,115 @@
+            question = Question()
+            question.addContext(f'data:image/png;base64,{doc.page_content}')
+            question.addQuestion(self.IGlobal._chat._prompt)
+
+            if self._cached_answer is not None:
+                # writeImage already called Gemini for this frame — reuse the result.
+                answer_text = self._cached_answer
+                self._cached_answer = None
+            else:
+                try:
+                    answer = self.IGlobal._chat.chat(question)
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)